WordPress/Classes/Models/BlockEditorSettings+GutenbergEditorSettings.swift

```diff
@@ -60,6 +73,16 @@ extension BlockEditorSettings {
             parsedElements.insert(BlockEditorSettingElement(fromRawRepresentation: gradient, type: .gradient, order: index, context: context))
         })
 
+        // Experimental Features
+        let galleryKey = BlockEditorExperimentalFeatureKeys.galleryRefactor.rawValue
+        let galleryRefactor = BlockEditorSettingElement(name: galleryKey,
```

Comment:
Since this flag is expected to be temporary I'm storing it in the `BlockEditorSettingElement` as a new type `experimentalFeatures`